### DISCUSSION TITLE
EMU License migration cost

### DISCUSSION THREAD

**Discussion by amojamo** [community/community#38781] [Enterprise]
### Select Topic Area

Question

### Body

Hi,

I've been trying to get in contact with Sales the past two - three weeks without luck. I'm really curious if GitHub really want to sell their products :)
 
But anyways, we were looking at EMU licenses and was told that there would be a migration cost of 12.5k USD if we were to upgrade from our current self-served Enterprise model to the sales-served EMU license model. This came as a surprise to us, and I wonder if there are other ways to circumvent the migration. Is it possible to create a new tenant in the "Sales serve EMU" model, and not migrating our current GitHub tenant?

Sincerely,
Adrian

**Comment by ossanna16** [comment]
Hi there @amojamo and welcome to our community! May I asked how you tried to engage with Sales? We are always trying to improve our processes so this would be helpful to know.

**Comment by amojamo** [comment]
Hi, sorry for the late reply. I've interacted with a sales representative from the Netherlands that wont answer my question for some reason. After not getting any replies from him, I've tried a couple of times to contact sales through the contact page, https://github.com/enterprise/contact without luck. After telling me about the additional cost of migration, he would not answer my additional question described in the original post^. 

-Adrian
